api/src/main/java/org/openmrs/Cohort.java

```diff
@@ -83,7 +85,8 @@ public Cohort(String name, String description, Patient[] patients) {
 		this(name, description, (Integer[]) null);
 		if (patients != null) {
 			for (Patient p : patients) {
-				memberIds.add(p.getPatientId());
+				CohortMember cohortMember = new CohortMember(this, p, new Date());
```

Comment:
CohortMember should have a constructor that takes just a cohort and patient instance and sets the start date to now by default that way you don't keep passing in new Date()